README.md

```diff
@@ -1 +1,175 @@
-# Hello-Microverse-Project-
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+  <br/>
+
+  <h3><b>Hello Microverse Project README File</b></h3>
+
+</div>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+# 📖 [Hello-Microverse] <a name="about-project"></a>
+
+**[your_project__name]** is a simple website that is made out of HTML and CSS. It dicplays the text "Hello Microverse!" on the screen, colored cyan in css.
+
+## 🛠 Built With <a name="built-with"></a>
+- HTML
+- CSS
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+> Describe between 1-3 key features of the application.
+
+- **[key_feature_1]** HTML
+- **[key_feature_2]** CSS
+- **[key_feature_3]** HTML, CSS Linters
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> Add a link to your deployed project.
+
+- [Live Demo Link](https://yourdeployedapplicationlink.com)
+
```

Comment:
- [ ] While you did a great job 😃, I see here that when I try to click on the link of the live demo to see the project I see that it is broken, so please try to put the correct one to let the visitors see the project or if you can not know how to add it you can write now ``` coming soon``` because you will learn in the future the GitHub pages to let you deploy your project 😊

Reply:
@osamaashraf6  I have also pushed my changes to the online repository.